### ISSUE TITLE
Luminosity contrast ratio of 'Edit fields' with background fails to meet the required ratio of 3:1.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#347]
Note: Same issue is observed wherever similar controls are present throughout the application.

Other instances:
1. Same issue is observed for the Data base name Dialog.
Issue: Luminosity contrast ratio of Ok and cancel buttons with background fails to meet the required ratio of 3:1.
Current ratio: 1.1:1.
Ex: Luminosity contrast ratio of Ok and cancel buttons with background should be greater than or equal to the minimum required ratio of 3:1.
Path: SqlNexus -> connect server dialog-> Navigate to Current DB dropdown in the header and select new data base-> Data base name Dialog gets displayed.
Note: Same issue repro for similar controls throughout the application.

2. Same issue is observed for the Data base name Dialog.
Issue: Luminosity contrast ratio of Edit fields and buttons with background fails to meet the required ratio of 3:1.
Current ration of Server name edit field is 1.1:1.
Current ration of Authentication edit field is 1.1:1.
Current ration of Username edit field is 1.1:1.
Current ration of Password edit field is 1.1:1.
Current ration of Connect button is 1.1:1.
Current ration of Cancel button is 1.1:1.
Ex: Luminosity contrast ratio of Edit fields and buttons with background should be greater than or equal to the minimum required ratio of 3:1.
Path: SqlNexus -> connect server dialog-> Navigate to the edit fields and buttons.

Impact on a Person with Disability (PWD):

Users who are color sensitive and have low vision are facing difficulty as Luminosity contrast ratio of 'Edit fields' with background fails to meet the required ratio 3:1.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Tools: Color Contrast Analyzer.

Pre-Requisites:
1. Install SQL NEXUS application on the desktop.
2. Turn on Color Contrast Analyzer.

Repro Steps:
Step 1: Open the SQL Nexus app on the desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: Navigate to SQL Perf Main and invoke it.
Step 4: Verify whether Luminosity contrast ratio of 'Edit fields' with background fails to meet the required ratio 3:1 or not.

Actual Result:
Luminosity contrast ratio of 'Edit fields' with background fails to meet the required ratio 3:1.
Current ratio:1.1:1

Expected Result: 
Luminosity contrast ratio of 'Edit fields' with background should be greater than or equal to the minimum required ratio of 3:1.
Ex: Visual borders should clearly be visible on all 4 sides, and it should meet the luminosity ratio of 3:1.
Note: Visually Broders can be provided for edit fields present in the header section.

Suggestive Fix:
When designing your site, try to be mindful of adjacent colors and those used to establish visual boundaries of controls, such as buttons, inputs, checkboxes, and radio buttons. This includes on focus indicators.​
This criterion can be easily solved manually as well, if you ensure that your default link and essential graphic styles observe a 3:1 contrast ratio. Use this contrast tool to check your ratio.​

**Comment by PiJoCoder** [comment]
Fixed by PR #365
